libgrease/src/state_machine/lifecycle.rs

```diff
@@ -339,10 +355,16 @@ pub mod test {
             },
             proof: b"my_update_proof".to_vec(),
         };
+        let mut rng = &mut rand::rng();
+        let (offset_self, statement_self) = circuits::make_keypair_ed25519_bjj_order(&mut rng);
+        let offset_self = Curve25519Secret::from_generic_scalar(&offset_self.into()).unwrap();
+        let (offset_peer, statement_peer) = circuits::make_keypair_ed25519_bjj_order(&mut rng);
+        let offset_peer = Curve25519Secret::from_generic_scalar(&offset_peer.into()).unwrap();
```

Comment:
Lifecycle is agnostic to where the data comes from, so better to hardcode some values here rather than depend on a dependent design domain